views/personal/residence.pug

```diff
@@ -29,4 +29,6 @@ block content
           option(value='Non Resident') #{__('Non Resident')}
     input#redirect(name='redirect', type='hidden', value='/personal/address')
 
+    input(type='hidden', name="_csrf", value=csrfToken)
```

Comment:
We could put this in the "formButtons" mixin, since every form has a button. 
We could also not because an input is not a button. 

What do you think @katedee?

Reply:
I think it might make sense to but it in formButtons, since this input is needed on every form page but only form pages. This would take the guesswork out of remembering whether you need it or not.

Reply:
I've added it to the mixin for now, I can always revert if we decide it's a bad idea.

Reply:
I think putting it there is a good idea, especially since it's on every page. We can always change the name of the mixin if it bothers anyone, but I don't really mind.